tests/codegen_testcases/solidity/unused_variable_elimination.sol

```diff
@@ -66,7 +66,7 @@ contract c {
         x = 102 + t*y/(t+5*y) + g + test3() - vec.push(2) + ct.sum(1, 2);
 		return 2;
 // CHECK: push array ty:int32[] value:int32 2
-// CHECK: _ = external call::regular address:%ct payload:%abi_encoded.temp.94 value:uint128 0 gas:uint64 0 accounts: seeds:
+// CHECK: return int32 2
```

Comment:
Isn't the call here anymore?
The purpose of this check it make sure the external call is not eliminated, when simplifying the expression in line 66.
Verifying that it returns 2 is not right here.

Reply:
Good catch, the call is still there, added the check.